Assign2/file_import_and_graph/main_effect.py

```diff
@@ -19,21 +20,29 @@ def main_effect(dict,var):
     import DOE_generator as deo_gen
 
     #Change the filepath in this line to reflect where the "DOE_options_only.csv" resides.
-    key_list,val_list,df = deo_gen.DOE_generator('C:/Users/jbeil/MDO_Project/Assign2/file_import_and_graph/DOE_options_only.csv')
```

Comment:
Same here. Can we get rid of "C:/Users/jbeil/MDO_Project/Assign2"

```suggestion
    key_list,val_list,df = deo_gen.DOE_generator('./file_import_and_graph/DOE_options_only.csv')
```

Reply:
Fixed